app/models/concerns/foreman_openscap/policy_extensions.rb

```diff
@@ -244,5 +251,28 @@ def populate_overrides(puppetclass, hostgroup)
         end
       end
     end
+
+    def set_certificates_overrides(puppetclass)
+      [CA_FILE_PARAMETER, HOST_CERTIFICATE_PARAMETER, HOST_PRIVATE_KEY_PARAMETER].each do |override_parameter|
+        unless certificate_param = puppetclass.class_params.where(:key => override_parameter).first
+          errors[:base] << _("Puppet class %{class} does not have %{parameter} class parameter.") % {:class => SCAP_PUPPET_CLASS, :parameter => override_parameter}
+          return false
+        end
+        certificate_param.override = true
+        case override_parameter
+          when CA_FILE_PARAMETER
+            certificate_param.default_value = '/etc/rhsm/ca/katello-server-ca.pem'
```

Comment:
Actually, subscription-manager config may be preferable, because it doesn't the variable substitution you'd have to deal with in rhsm.conf,

   repo_ca_cert = /etc/rhsm/ca/katello-server-ca.pem

So, yea, that could be turned into a fact